src/prob/opf_iv.jl

```diff
@@ -1,53 +1,40 @@
 ""
 function run_opf_iv(file, model_constructor, optimizer; kwargs...)
-    return run_model(file, model_constructor, optimizer, post_opf_iv, solution_builder=solution_opf_iv!; kwargs...)
+    # return run_model(file, model_constructor, optimizer, post_opf_iv, solution_builder=solution_opf_iv!; kwargs...)
+    return run_model(file, model_constructor, optimizer, post_opf_iv; kwargs...)
```

Comment:
Default solution builder works becouse of expressions in `pm.var`?

Reply:
indeed